runtime/gc_vlhgc/SchedulingDelegate.cpp

```diff
@@ -629,12 +633,31 @@ MM_SchedulingDelegate::updateLiveBytesAfterPartialCollect()
 			_liveSetBytesAfterPartialCollect += region->getSize();
 			_liveSetBytesAfterPartialCollect -= memoryPool->getActualFreeMemorySize();
 			_liveSetBytesAfterPartialCollect -= memoryPool->getDarkMatterBytes();
-		} else if (region->isArrayletLeaf()) {
+		} else if (region->isArrayletLeaf() && !_extensions->isVirtualLargeObjectHeapEnabled) {
 			if (_extensions->objectModel.isObjectArray(region->_allocateData.getSpine())) {
 				_liveSetBytesAfterPartialCollect += region->getSize();
 			}
 		} 
 	}
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+	if (_extensions->isVirtualLargeObjectHeapEnabled) {
+		MM_SparseVirtualMemory *largeObjectVirtualMemory = _extensions->largeObjectVirtualMemory;
+		J9HashTableState walkState;
+
+		MM_SparseDataTableEntry *sparseDataEntry = (MM_SparseDataTableEntry *)hashTableStartDo(largeObjectVirtualMemory->getSparseDataPool()->getObjectToSparseDataTable(), &walkState);
+		const uintptr_t arrayletLeafSize = _extensions->getOmrVM()->_arrayletLeafSize;
+		while (NULL != sparseDataEntry) {
+			J9Object *spineObject = (J9Object *)sparseDataEntry->_proxyObjPtr;
+
+			if (_extensions->objectModel.isObjectArray(spineObject)) {
+//				_liveSetBytesAfterPartialCollect += sparseDataEntry->_size;
+				_liveSetBytesAfterPartialCollect += MM_Math::roundToCeiling(arrayletLeafSize, sparseDataEntry->_size);
```

Comment:
i'm fine with either ceiling or not even now, but if you do ceiling now, don't forget to remove it when we start sharing arraylet regions